minard/templates/noise.html

```diff
@@ -25,7 +25,7 @@
       </tr>
     </thead>
     <tbody>
-      {% for run in runs|reverse if run %}
+      {% for run in runs if run %}
```

Comment:
If you want a guaranteed order you should include an `ORDER BY` expression in your SQL statements. I think the general style on minard is to have newest things at the top, so you can either `ORDER BY run_number ASC` and then reverse here or `ORDER BY run_number DESC` and then not reverse here.